contracts/TalentLayerPlatformID.sol

```diff
@@ -151,6 +151,7 @@ contract TalentLayerPlatformID is ERC721Upgradeable, AccessControlUpgradeable, U
         // Increment counter to start platform ids at index 1
         nextPlatformId.increment();
         mintStatus = MintStatus.ONLY_WHITELIST;
+        emit ArbitratorAdded(address(0), false);
```

Comment:
so this will not work as initialize will not be called after an upgrade, we need to create a new V2 here. Check doc https://docs.openzeppelin.com/contracts/4.x/api/proxy#Initializable

Reply:
Removed the line